fl4health/server/base_server.py

```diff
@@ -132,6 +178,15 @@ def evaluate_round(
             if loss_aggregated:
```

Comment:
I know this isn't your code, but one line 177, I think you can replace `(results, failures)` with just `_`